pkg/cloudprovider/providers/vsphere/nodemanager.go

```diff
@@ -176,13 +176,13 @@ func (nm *NodeManager) DiscoverNode(node *v1.Node) error {
 				defer cancel()
 				vm, err := res.datacenter.GetVMByUUID(ctx, nodeUUID)
 				if err != nil {
-					glog.V(4).Infof("Error %q while looking for vm=%+v in vc=%s and datacenter=%s",
-						err, node.Name, vm, res.vc, res.datacenter.Name())
+					glog.V(4).Infof("Error %q while looking for node %s in vc=%s and datacenter=%s",
+						err, node.Name, res.vc, res.datacenter.Name())
```

Comment:
@rsc 
please, change this one to:

```
glog.V(4).Infof("Error while looking for vm=%+v in vc=%s and datacenter=%s: %v",
	vm, res.vc, res.datacenter.Name(), err)
```
/cc @frapposelli 
/cc @kubernetes/sig-cloud-provider-pr-reviews

Reply:
👍 we need to report the `vm` reference to the user for troubleshooting.